backside/places/handler.go

```diff
@@ -3,18 +3,23 @@ package places
 import (
 	"encoding/json"
 	"net/http"
-	"sk8.town/backside/errs"
 
+	"github.com/gorilla/schema"
+
+	"sk8.town/backside/errs"
 	"sk8.town/backside/places/dto"
 )
 
+var decoder = schema.NewDecoder()
```

Comment:
> Set a Decoder instance as a package global, because it caches
  meta-data about structs, and an instance can be shared safely.

source: https://pkg.go.dev/github.com/gorilla/schema#section-readme